azurerm/helpers/validate/strings.go

```diff
@@ -18,3 +18,27 @@ func NoEmptyStrings(i interface{}, k string) ([]string, []error) {
 
 	return nil, nil
 }
+
+// PrivateLinkEnpointRequestMessage validates that the Private Link Enpoint Request Message is less than 140 characters
+func PrivateLinkEnpointRequestMessage(i interface{}, k string) (_ []string, errors []error) {
+	return stringMaxLength(140)(i, k)
+}
+
+func stringMaxLength(maxLength int) func(i interface{}, k string) (_ []string, errors []error) {
+	return func(i interface{}, k string) (_ []string, errors []error) {
+		v, ok := i.(string)
+		if !ok {
+			return nil, []error{fmt.Errorf("expected type of %q to be string", k)}
+		}
+
+		if len(v) > maxLength {
+			return nil, []error{fmt.Errorf("%q must not be longer than %d characters, got %d", k, maxLength, len(v))}
+		}
+
+		if strings.TrimSpace(v) == "" {
+			return nil, []error{fmt.Errorf("%q must not be empty", k)}
+		}
+
+		return
+	}
+}
```

Comment:
We have `validation.StringLenBetween` that should work instead?

Reply:
`stringLenBetween` is a private function which is why I wrote the wrapper. I didn't want to make it public out of fear of breaking other code.

Reply:
It is? looking at the code:
`ValidateFunc: validation.StringLenBetween(1, 1024),`